docs/azure_arc.md

```diff
@@ -0,0 +1,48 @@
+# Deploying to Azure Arc
+
+This template supports deployment to Azure Arc. The following document lists the requirements to setup Azure Arc to support the template, as well as how you deploy the application to an Arc-enable Kubernetes Cluster.
+
+- [Deploying to Azure Arc](#deploying-to-azure-arc)
+  - [Prerequisites](#prerequisites)
+  - [Regions and resource group support](#regions-and-resource-group-support)
+    - [Azure Arc enabled App Service](#azure-arc-enabled-app-service)
+
+## Prerequisites
+
+In order to deploy this template to Azure Arc, you need to have the Arc-enabled Kubernetes Cluster created. This template assumes this has already been done. To learn more about configuring an Arc environment for App Service, please see this [blog](https://aka.ms/ArcEnabledAppServices-Build2021-Blog).
+
+It's also requred to have Arc-enabled dataservices Data Controller installed in the Arc Cluster, please see more [here](https://docs.microsoft.com/en-us/azure/azure-arc/data/create-data-controller). This repository assumes a directly connected data controller.
+
+## Regions and resource group support
+
+[Please check here for latest information](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster) 
```

Comment:
Can we explain why we are asking the user to check for latest info?

Reply:
👍🏼